source/game/kart/KartMove.hh

```diff
@@ -47,6 +47,7 @@ public:
     void controlOutsideDriftAngle();
     void calcRotation();
     void calcVehicleSpeed();
+    void decelerate();
```

Comment:
This does not follow the naming convention for all the other functions in this TU. Can you rename to `calcDeceleration`?